packages/react-ui/components/Button/__stories__/ButtonNEW.stories.tsx

```diff
@@ -0,0 +1,435 @@
+import React from 'react';
+import ArchivePack from '@skbkontur/react-icons/ArchivePack';
+import OkIcon from '@skbkontur/react-icons/Ok';
+import SearchIcon from '@skbkontur/react-icons/Search';
+import { CreeveyTests } from 'react-ui/typings/stories';
```

Comment:
Здесь автоматически вставился импорт из 'react-ui'. Он невалидный. Его нужно поменять на относительный путь до этого файла. 

```suggestion
import { CreeveyTests } from '../../../typings/stories';
```